create_gazebo_plugins/src/gazebo_ros_create.cpp

```diff
@@ -100,7 +100,7 @@ void GazeboRosCreate::Load(physics::ModelPtr _parent, sdf::ElementPtr _sdf )
     if (!base_geom_)
     {
       ROS_ERROR("Unable to find geom[%s]",base_geom_name_.c_str());
-      return;
+      // return;
```

Comment:
Could you add a comment here as to why it's commented out? It seems like this catch was covering up an error condition in the past, and the original in line 93 should be working and has a parameter wrong maybe?

Actually looking at it base_geom_ does not appear to be used at all. It's commented in line 107. Can we just remove 90-107 entirely? (and the declaration of base_geom_ in the header) A search only finds it referenced in those two places: https://github.com/turtlebot/turtlebot_create_desktop/search?utf8=%E2%9C%93&q=base_geom_